test/common/fit_common.py

```diff
@@ -473,9 +471,13 @@ def remote_shell(shell_cmd, expect_receive="", expect_send="", timeout=300,
 
 
 def scp_file_to_ora(src_file_name):
+    # legacy call
```

Comment:
@stuart-stanley  Because I have stories to update these files separately. They all need work.